### PR TITLE
PR for 09--hello-javascript

### PR DESCRIPTION
**Pull request by mfees**
https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/120

### CODE REVIEW

**Comment by carolinapowers** [comment]
:+1:

**Comment by johnalxndr** [comment]
:+1:

**Comment by adamkarb** [comment]
:+1:

**Comment by mfees** [comment]
@adamkarb am I your first thumb?!

**Comment by adamkarb** [comment]
@mfees I think there was someone before you but can't remember

**Comment by mfees** [comment]
well the magic is just gone now

**Comment by sak55** [comment]
I may not be the first but i'm the next one :+1:

**Comment by mfees** [comment]
SO MUCH LOVE <3
@carolinapowers 
@johnalxndr 
@adamkarb 
@sak55

**Comment by jgarcia99** [comment]
:+1:

File: docs/js.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+```javascript
+50 < 100 == true
+50 > 100 == false
+
+('meh' != meh) == true
+```
+
+### [Primative: Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)
+
+*_symbol_: 'The numeric value of the object being created.'
```
Comment (al-the-x) [inline docs/js.md:24]: Might there be additional symbols that we can use to represent numbers? You might at least list the "numeric values" for reference. You should have more to add here after today's lecture. To exponents and beyond!

File: docs/js.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+```javascript
+1
+2
+3
+10
+```
+
+### [Primative: String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+
+*_symbol_: 'a sequence of characters'
```
Comment (al-the-x) [inline docs/js.md:36]: Well, a sequence of characters _surrounded_ by single (`'`) or double (`"`) quotes. Any _other_ way to create a `String` value?

File: docs/js.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+```
+
+### [Built in Values: NaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'NaN'
+*_pronunciation_: "not a number", "nahn"
+*_examples_:
+```javascript
+"cat" == NaN
+```
```
Comment (al-the-x) [inline docs/js.md:59]: So, I'm not sure this comparison holds true... Be sure to run your examples through the REPL or `node` command to confirm! You don't want to document a _bad_ example.

File: docs/js.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+### [Operator: Decrement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: '--'
+*_pronunciation_: "decrement"
+*_example_:
+```javascript
+12-- == 11
+```
```
Comment (al-the-x) [inline docs/js.md:107]: For both the increment and decrement operator, you'll run into some issues running those examples. These are both _Assignment Operators_ and, as such, can only be used with values stored in variables, e.g.

``` javascript
var A = 10, B = -10;

console.log(--A); // prints 9
console.log(B++); // prints -10
console.log(B); // prints -9...?
```

File: docs/js.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+### [Operator: Unary Negation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: `-`
+*_pronunciation_: "unary negation"
+*_example_:
+```javascript
+If x is 10, then -x returns -10
+```
```
Comment (al-the-x) [inline docs/js.md:116]: So, technically not JavaScript... Maybe using `assert` could help here?

``` javascript
var x = 10;
assert(-x === -10);
```

File: docs/js.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+### [Operator: Assignment %=](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Assignment_operators)
+
+*_symbol_: '%='
+*_pronunciation_: "equal x modulus"
+*_example_:
+```javascript
+x %= y == x = x % y 
+```
```
Comment (al-the-x) [inline docs/js.md:161]: Hey, good guess! Asking a question like this on [Stack Overflow](http://stackoverflow.com) (or just searching for it first, can get you trolled but also might get you a decent answer.

**Pull request merged**